torch/csrc/jit/tensorexpr/loopnest.h

```diff
@@ -558,6 +558,15 @@ class TORCH_API LoopNest {
   }
   std::vector<BufPtr> getIntermediateBufs() const;
 
+  // Finds which is the outer for loop between a and b for loops. Returns
+  // nullptr if they don't have same parent.
```

Comment:
This function returns `nullptr` when neither of the given 2 Fors is an ancestor of the other. Not having the same parent is not being checked here. Actually, if the given 2 Fors have a common parent, this will return `nullptr`, since in that case, neither can be a parent of the other.

Reply:
Thanks for catching the bad documentation. Updated it.